src/profile-logic/flame-graph.js

```diff
@@ -89,16 +89,17 @@ export function getFlameGraphTiming(
     timeOffset[depth + 1] = timeOffset[depth];
     timeOffset[depth] += totalTimeRelative;
 
-    let children = callTree.getChildren(nodeIndex).slice();
+    const children = callTree.getChildren(nodeIndex).slice();
     children.sort(
       (a, b) =>
         callTree.getNodeData(a).funcName < callTree.getNodeData(b).funcName
           ? 1
           : -1
     );
-    children = children.map(nodeIndex => ({ nodeIndex, depth: depth + 1 }));
 
-    stack.push(...children);
+    for (let i = 0; i < children.length; i++) {
+      stack.push({ nodeIndex: children[i], depth: depth + 1 });
+    }
```

Comment:
(sorry for the stupid comments as I discover this code now :) ).

I see we're sorting in descending order, and adding the children here, because we use "stack.pop()" above. Maybe we can sort in ascending order and add the children in reverse order here. Not a big difference.
_But_ at least I'd like some comment where we add it in reverse to explain why we do that.

Reply:
Great comments, not stupid :-)

Yes, good idea. I reversed the sort and push and added a comment explaining it.